code/modules/surgery/implant_removal.dm

```diff
@@ -1,81 +1,199 @@
 /datum/surgery/implant_removal
-	name = "Implant removal"
+	name = "Implant extraction/destruction"
 	possible_locs = list(BODY_ZONE_CHEST)
 	steps = list(
 		/datum/surgery_step/incise,
 		/datum/surgery_step/clamp_bleeders,
 		/datum/surgery_step/retract_skin,
-		/datum/surgery_step/extract_implant,
+		/datum/surgery_step/search_for_implants,
+		/datum/surgery_step/destroy_extract_implant,
 		/datum/surgery_step/close,
 	)
+	var/list/discovered_implants = list()
 
-//extract implant
-/datum/surgery_step/extract_implant
-	name = "extract implant (hemostat)"
+/datum/surgery_step/search_for_implants
+	name = "search for implants (hand/t-ray)"
```

Comment:
This doesn't mention health analyzers at all. I would make it say (health analyzer/hand)